InventoryWatch/SettingsView.swift

```diff
@@ -86,8 +87,10 @@ struct SettingsView: View {
                         Text("Only show results for preferred models")
                             .padding(.leading, 4)
                     }
-                    
-                    
+                    Toggle(isOn: $showInMenuBar) {
+                        Text("Show in Menu Bar (requires Mac OS Ventura)")
+                            .padding(.leading, 4)
+                    }
```

Comment:
to try and keep the Settings window balanced, can you move this toggle to the other `VStack`, ~on line `111`? Probably can remove the `padding` as well.